File: Python/Praktikum2-5.py
import math
B=float(input("B = "))
A=float(input("A = "))
C=math.sqrt(B*B-A*A)
print("Alas = ",round(C),("cm"))
print("Tinggi =",round(A),("cm"))
print("Keliling = ", round(A+B+C),("cm"))
print("Luas = ",round(A*C/2),("cm^2"))

File: C/Praktikum2-5.c
#include <stdio.h>
#include<math.h>

int main(){
    float A, B;
    printf("B= ");scanf("%f", &B);
    printf("A= ");scanf("%f", &A);
    float C = sqrt(B*B-A*A);
    float keliling= A+B+C;
    float luas= A*C/2;
    printf("Alas =%.f cm\n",C);
    printf("Tinggi = %.f cm\n", A);
    printf("Keliling = %.f cm\n",keliling);
    printf("Luas = %.f cm^2",luas);
}